hw/ip/kmac/data/kmac.hjson

```diff
@@ -126,6 +126,12 @@
       randcount: "64"
       randtype:  "perm"
     }
+    { name:      "RndCnstStoragePerm"
+      desc:      "Compile-time random permutation for Storage Entropy"
+      type:      "kmac_pkg::storage_perm_t"
+      randcount: "1600"
+      randtype:  "perm"
```

Comment:
this will get exposed to the top level right? this is probably something we want nuvoton to regenerate.

Reply:
Ah.. yep. I should re-generate top here too.
A few more random constants for KMAC has been added. I will send an email to them

Reply:
CC: @arnonsha